rust/item/src/utils.rs

```diff
@@ -1826,3 +1803,22 @@ pub fn get_item_usage(args: GetItemUsageArgs) -> Result<ItemUsage> {
 
     Ok(item_usage)
 }
+
+// returns true if the namespace program called the item program
+pub fn is_namespace_program_caller(ixns: &AccountInfo) -> bool {
+    let current_index =
+        anchor_lang::solana_program::sysvar::instructions::load_current_index_checked(&ixns)
+            .unwrap() as usize;
+    let current_ixn =
+        anchor_lang::solana_program::sysvar::instructions::load_instruction_at_checked(
+            current_index,
+            &ixns,
+        )
+        .unwrap();
```

Comment:
What is the difference between `get_instruction_relative` and this?

Reply:
I'm not sure, this is how I've always done it, but ill check out that function.

Reply:
From the docs it looks like they're nearly identical, one is relative to current ix, one is absolute index in regards to entire tx

Reply:
nvm, don't bother merge.